### ISSUE TITLE
Problem using IQKeyboardReturnHandler

### ISSUE THREAD

**Issue by tedrog36** [hackiftekhar/IQKeyboardManager#402]
Hi,

I wired in the IQKeyboardReturnHandler hoping that the handling of next/done return key (returnKeyType = UIReturnKeyNext or UIReturnKeyDone) would get handled the same as the arrow keys in the keyboard toolbar.  That doesn't seem to be working.  I am also using  addPreviousNextDoneOnKeyboardWithTarget:previousAction:nextAction:doneAction:.  I was assuming once I started using IQKeyboardReturnHandler that my previousAction, nextAction, and doneAction would get call when hitting the return key but they do not.  Am I doing something wrong or is that not how it is supposed to work?

Ted

**Comment by tedrog36** [comment]
I discovered part of my issue.  My textFields are in a table that is not yet populated in viewDidLoad so calling so [[IQKeyboardReturnKeyHandler alloc] initWithViewController:self] did not find any text fields.  Duh!   I tried calling addTextFieldView when the cells were created but still didn't get the behavior I was expecting which was for my nextAction and doneAction to get called.
Anyway, I just handle the textFieldShouldReturn and handle the next/prev that way, so things are working for me now.
